src/CssInliner.php

```diff
@@ -1276,10 +1283,17 @@ protected function addStyleElementToDocument(string $css): void
      * This method assumes that there always is a HEAD element.
      *
      * @return \DOMElement
+     *
+     * @throws \UnexpectedValueException
      */
     private function getHeadElement(): \DOMElement
     {
-        return $this->domDocument->getElementsByTagName('head')->item(0);
+        $node = $this->getDomDocument()->getElementsByTagName('head')->item(0);
+        if (!$node instanceof \DOMElement) {
+            throw new \UnexpectedValueException('The head is no DOMElement. This should never happen.', 1617923227);
```

Comment:
Typo.  Should be "is not a DOMElement"?  Also, this exception could occur if there is no `<head>` element (as we have seen in #923 and #957) so the message should be something like "There is no head element or it is not a DOMElement. ..."

Reply:
Actually I think the if the exception occurs it will always be because there is no `<head>` element.  If there is one, it will necessarily be a `DOMElement`.

Reply:
I've updated the message accordingly.